src/game-item.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { ChainID } from '@harmony-js/utils'
+import { expect, use } from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import sinon from 'sinon'
+import { GameItems } from './game-item'
+import { HarmonyShards } from './interfaces'
+import { PrivateKey } from './private-key'
+import { HRC1155_CONTRACT_ADDRESS, TEST_ACCOUNT_1 } from './tests/constants'
+import { ABI } from './tests/contracts/GameItems/abi'
+
+describe('Game Item Provider', () => {
+  use(chaiAsPromised)
+
+  let contract: GameItems
+  let provider: PrivateKey
+
+  before(() => {
+    provider = new PrivateKey(HarmonyShards.SHARD_0_TESTNET, TEST_ACCOUNT_1.privateKey, ChainID.HmyTestnet)
```

Comment:
You can use a [Key](https://github.com/blockcoders/harmony-marketplace-sdk/blob/main/src/key.ts#L10) w/o any pk, because you don't need to sign anything to call the queries.

Reply:
Oh, that's okay!